### ISSUE TITLE
Changing Issues to Tasks

### ISSUE THREAD

**Issue by whotopia** [ishikawa999/redmine_message_customize#59]
This is a fantastic tool you have written.  To anyone who might need it, [redmine_message_customize](https://github.com/farend/redmine_message_customize) plugin, the following changes 'Issues' to 'Tasks' and 'Versions' to 'Milestones'  across all of the locale oriented text that I have been able to find for Redmine and all my plugins.   Note you may need to add more for other languages / locales as required.

Hopefully useful for someone else:

```
---
en:
  label_issues_panel_plural: Tasks Panel
  project_module_issues_panel: Tasks Panel
  notice_unable_delete_version: Unable to delete milestone.
  error_can_not_reopen_issue_on_closed_version: An task assigned to a closed version
    cannot be reopened
  field_fixed_version: Target milestone
  field_version: Milestone
  field_default_version: Default milestone
  permission_manage_versions: Manage milestones
  label_issue_fixed_version_updated: Target milestone updated
  label_roadmap: Milestones
  label_version: Milestone
  label_version_new: New milestone
  label_version_plural: Milestones
  label_version_and_files: Milestones (%{count}) and Files
  label_close_versions: Close completed milestones
  label_roadmap_no_issues: No issues for this version
  label_current_version: Current milestone
  label_show_completed_versions: Show completed milestones
  label_completed_versions: Completed milestones
  label_attribute_of_fixed_version: Target milestone's %{name}
  label_latest_compatible_version: Latest compatible milestone
  button_rollback: Rollback to this milestone
  activerecord:
    errors:
      messages:
        cant_link_an_issue_with_a_descendant: An task cannot be linked to one of its
          subtasks
        earlier_than_minimum_start_date: cannot be earlier than %{date} because of
          preceding tasks
        open_issue_with_closed_parent: An open task cannot be attached to a closed
          parent task
  notice_failed_to_save_issues: 'Failed to save %{count} task(s) on %{total} selected:
    %{ids}.'
  notice_issue_done_ratios_updated: Task done ratios updated.
  notice_issue_successful_create: Task %{id} created.
  notice_issue_update_conflict: The task has been updated by an other user while you
    were editing it.
  notice_issue_not_closable_by_open_tasks: This task cannot be closed because it has
    at least one open subtask.
  notice_issue_not_closable_by_blocking_issue: This task cannot be closed because
    it is blocked by at least one open issue.
  notice_issue_not_reopenable_by_closed_parent_issue: This task cannot be reopened
    because its parent issue is closed.
  error_issue_not_found_in_project: The task was not found or does not belong to this
    project
  error_no_default_issue_status: No default task status is defined. Please check your
    configuration (Go to "Administration -> Task statuses").
  error_can_not_delete_tracker: This tracker contains tasks and cannot be deleted.
  error_issue_done_ratios_not_updated: Task done ratios not updated.
  error_unable_delete_issue_status: Unable to delete task status (%{value})
  error_no_tracker_allowed_for_new_issue_in_project: The project doesn't have any
    trackers for which you can create an task
  error_no_projects_with_tracker_allowed_for_new_issue: There are no projects with
    trackers for which you can create an task
  error_cannot_reassign_time_entries_to_an_issue_about_to_be_deleted: Spent time cannot
    be reassigned to an task that is about to be deleted
  mail_subject_reminder: "%{count} task(s) due in the next %{days} days"
  mail_body_reminder: "%{count} task(s) that are assigned to you are due in the next
    %{days} days:"
  field_issue: Task
  field_is_closed: Task closed
  field_is_in_roadmap: Tasks displayed in roadmap
  field_issue_to: Related task
  field_assignable: Tasks can be assigned to users with this role
  field_issues_visibility: Tasks visibility
  field_history_default_tab: Task's history default tab
  setting_issues_export_limit: Tasks export limit
  setting_issue_list_default_columns: Tasks list defaults
  setting_issue_done_ratio: Calculate the task done ratio with
  setting_issue_done_ratio_issue_field: Use the task field
  setting_issue_done_ratio_issue_status: Use the task status
  setting_issue_group_assignment: Allow task assignment to groups
  setting_default_issue_start_date_to_creation_date: Use current date as start date
    for new tasks
  setting_commit_cross_project_ref: Allow tasks of all the other projects to be referenced
    and fixed
  setting_link_copied_issue: Link tasks on copy
  setting_close_duplicate_issues: Close duplicate tasks automatically
  setting_show_status_changes_in_mail_subject: Show status changes in task mail notifications
    subject
  permission_manage_categories: Manage task categories
  permission_view_issues: View Tasks
  permission_add_issues: Add tasks
  permission_edit_issues: Edit tasks
  permission_edit_own_issues: Edit own tasks
  permission_copy_issues: Copy tasks
  permission_manage_issue_relations: Manage task relations
  permission_set_issues_private: Set tasks public or private
  permission_set_own_issues_private: Set own tasks public or private
  permission_delete_issues: Delete tasks
  permission_manage_related_issues: Manage related tasks
  permission_import_issues: Import tasks
  project_module_issue_tracking: Task tracking
  label_issue: Task
  label_issue_new: New task
  label_issue_plural: Tasks
  label_issue_view_all: View all tasks
  label_issues_by: Tasks by %{value}
  label_issue_added: Task added
  label_issue_updated: Task updated
  label_issue_status: Task status
  label_issue_status_plural: Task statuses
  label_issue_category: Task category
  label_issue_category_plural: Task categories
  label_reported_issues: Reported tasks
  label_assigned_issues: Assigned tasks
  label_assigned_to_me_issues: Tasks assigned to me
  label_updated_issues: Updated tasks
  label_x_issues:
    zero: 0 tasks
    one: 1 task
    other: "%{count} tasks"
  label_any_issues_in_project: any tasks in project
  label_any_issues_not_in_project: any tasks not in project
  label_no_issues_in_project: no tasks in project
  label_any_open_issues: any open tasks
  label_no_open_issues: no open tasks
  label_issue_tracking: Task tracking
  label_watched_issues: Watched tasks
  label_related_issues: Related tasks
  label_bulk_edit_selected_issues: Bulk edit selected tasks
  label_user_mail_notify_about_high_priority_issues_html: Also notify me about tasks
    with a priority of <em>%{prio}</em> or higher
  label_update_issue_done_ratios: Update task done ratios
  label_issues_visibility_all: All tasks
  label_issues_visibility_public: All non private tasks
  label_issues_visibility_own: Tasks created by or assigned to the user
  label_attribute_of_issue: Task's %{name}
  label_link_copied_issue: Link copied task
  label_search_open_issues_only: Open tasks only
  label_import_issues: Import tasks
  label_new_project_issue_tab_enabled: Display the "New task" tab
  text_tip_issue_begin_day: task beginning this day
  text_tip_issue_end_day: task ending this day
  text_tip_issue_begin_end_day: task beginning and ending this day
  text_issues_ref_in_commit_messages: Referencing and fixing tasks in commit messages
  text_issue_added: Task %{id} has been reported by %{author}.
  text_issue_updated: Task %{id} has been updated by %{author}.
  text_issue_category_destroy_question: Some tasks (%{count}) are assigned to this
    category. What do you want to do?
  text_issue_category_reassign_to: Reassign tasks to this category
  text_no_configuration_data: |-
    Roles, trackers, task statuses and workflow have not been configured yet.
    It is highly recommended to load the default configuration. You will be able to modify it once loaded.
  text_issues_destroy_confirmation: Are you sure you want to delete the selected task(s)?
  text_destroy_time_entries_question: "%{hours} hours were reported on the tasks you
    are about to delete. What do you want to do?"
  text_reassign_time_entries: 'Reassign reported hours to this task:'
  enumeration_issue_priorities: Task priorities
  description_issue_category_reassign: Choose task category
en-GB:
  notice_unable_delete_version: Unable to delete milestone.
  error_can_not_reopen_issue_on_closed_version: An task assigned to a closed version
    cannot be reopened
  field_fixed_version: Target milestone
  field_version: Milestone
  field_default_version: Default milestone
  permission_manage_versions: Manage milestones
  label_issue_fixed_version_updated: Target milestone updated
  label_roadmap: Milestones
  label_version: Milestone
  label_version_new: New milestone
  label_version_plural: Milestones
  label_version_and_files: Milestones (%{count}) and Files
  label_close_versions: Close completed milestones
  label_roadmap_no_issues: No issues for this version
  label_current_version: Current milestone
  label_show_completed_versions: Show completed milestones
  label_completed_versions: Completed milestones
  label_attribute_of_fixed_version: Target milestone's %{name}
  label_latest_compatible_version: Latest compatible milestone
  button_rollback: Rollback to this milestone
  activerecord:
    errors:
      messages:
        cant_link_an_issue_with_a_descendant: An task cannot be linked to one of its
          subtasks
        earlier_than_minimum_start_date: cannot be earlier than %{date} because of
          preceding tasks
        open_issue_with_closed_parent: An open task cannot be attached to a closed
          parent task
  notice_failed_to_save_issues: 'Failed to save %{count} task(s) on %{total} selected:
    %{ids}.'
  notice_issue_done_ratios_updated: Task done ratios updated.
  notice_issue_successful_create: Task %{id} created.
  notice_issue_update_conflict: The task has been updated by an other user while you
    were editing it.
  notice_issue_not_closable_by_open_tasks: This task cannot be closed because it has
    at least one open subtask.
  notice_issue_not_closable_by_blocking_issue: This task cannot be closed because
    it is blocked by at least one open issue.
  notice_issue_not_reopenable_by_closed_parent_issue: This task cannot be reopened
    because its parent issue is closed.
  error_issue_not_found_in_project: The task was not found or does not belong to this
    project
  error_no_default_issue_status: No default task status is defined. Please check your
    configuration (Go to "Administration -> Task statuses").
  error_can_not_delete_tracker: This tracker contains tasks and cannot be deleted.
  error_issue_done_ratios_not_updated: Task done ratios not updated.
  error_unable_delete_issue_status: Unable to delete task status (%{value})
  error_no_tracker_allowed_for_new_issue_in_project: The project doesn't have any
    trackers for which you can create an task
  error_no_projects_with_tracker_allowed_for_new_issue: There are no projects with
    trackers for which you can create an task
  error_cannot_reassign_time_entries_to_an_issue_about_to_be_deleted: Spent time cannot
    be reassigned to an task that is about to be deleted
  mail_subject_reminder: "%{count} task(s) due in the next %{days} days"
  mail_body_reminder: "%{count} task(s) that are assigned to you are due in the next
    %{days} days:"
  field_issue: Task
  field_is_closed: Task closed
  field_is_in_roadmap: Tasks displayed in roadmap
  field_issue_to: Related task
  field_assignable: Tasks can be assigned to users with this role
  field_issues_visibility: Tasks visibility
  field_history_default_tab: Task's history default tab
  setting_issues_export_limit: Tasks export limit
  setting_issue_list_default_columns: Tasks list defaults
  setting_issue_done_ratio: Calculate the task done ratio with
  setting_issue_done_ratio_issue_field: Use the task field
  setting_issue_done_ratio_issue_status: Use the task status
  setting_issue_group_assignment: Allow task assignment to groups
  setting_default_issue_start_date_to_creation_date: Use current date as start date
    for new tasks
  setting_commit_cross_project_ref: Allow tasks of all the other projects to be referenced
    and fixed
  setting_link_copied_issue: Link tasks on copy
  setting_close_duplicate_issues: Close duplicate tasks automatically
  setting_show_status_changes_in_mail_subject: Show status changes in task mail notifications
    subject
  permission_manage_categories: Manage task categories
  permission_view_issues: View Tasks
  permission_add_issues: Add tasks
  permission_edit_issues: Edit tasks
  permission_edit_own_issues: Edit own tasks
  permission_copy_issues: Copy tasks
  permission_manage_issue_relations: Manage task relations
  permission_set_issues_private: Set tasks public or private
  permission_set_own_issues_private: Set own tasks public or private
  permission_delete_issues: Delete tasks
  permission_manage_related_issues: Manage related tasks
  permission_import_issues: Import tasks
  project_module_issue_tracking: Task tracking
  label_issue: Task
  label_issue_new: New task
  label_issue_plural: Tasks
  label_issue_view_all: View all tasks
  label_issues_by: Tasks by %{value}
  label_issue_added: Task added
  label_issue_updated: Task updated
  label_issue_status: Task status
  label_issue_status_plural: Task statuses
  label_issue_category: Task category
  label_issue_category_plural: Task categories
  label_reported_issues: Reported tasks
  label_assigned_issues: Assigned tasks
  label_assigned_to_me_issues: Tasks assigned to me
  label_updated_issues: Updated tasks
  label_x_issues:
    zero: 0 tasks
    one: 1 task
    other: "%{count} tasks"
  label_any_issues_in_project: any tasks in project
  label_any_issues_not_in_project: any tasks not in project
  label_no_issues_in_project: no tasks in project
  label_any_open_issues: any open tasks
  label_no_open_issues: no open tasks
  label_issue_tracking: Task tracking
  label_watched_issues: Watched tasks
  label_related_issues: Related tasks
  label_bulk_edit_selected_issues: Bulk edit selected tasks
  label_user_mail_notify_about_high_priority_issues_html: Also notify me about tasks
    with a priority of <em>%{prio}</em> or higher
  label_update_issue_done_ratios: Update task done ratios
  label_issues_visibility_all: All tasks
  label_issues_visibility_public: All non private tasks
  label_issues_visibility_own: Tasks created by or assigned to the user
  label_attribute_of_issue: Task's %{name}
  label_link_copied_issue: Link copied task
  label_search_open_issues_only: Open tasks only
  label_import_issues: Import tasks
  label_new_project_issue_tab_enabled: Display the "New task" tab
  text_tip_issue_begin_day: task beginning this day
  text_tip_issue_end_day: task ending this day
  text_tip_issue_begin_end_day: task beginning and ending this day
  text_issues_ref_in_commit_messages: Referencing and fixing tasks in commit messages
  text_issue_added: Task %{id} has been reported by %{author}.
  text_issue_updated: Task %{id} has been updated by %{author}.
  text_issue_category_destroy_question: Some tasks (%{count}) are assigned to this
    category. What do you want to do?
  text_issue_category_reassign_to: Reassign tasks to this category
  text_no_configuration_data: |-
    Roles, trackers, task statuses and workflow have not been configured yet.
    It is highly recommended to load the default configuration. You will be able to modify it once loaded.
  text_issues_destroy_confirmation: Are you sure you want to delete the selected task(s)?
  text_destroy_time_entries_question: "%{hours} hours were reported on the tasks you
    are about to delete. What do you want to do?"
  text_reassign_time_entries: 'Reassign reported hours to this task:'
  enumeration_issue_priorities: Task priorities
  description_issue_category_reassign: Choose task category

```

**Comment by ishikawa999** [comment]
Thanks for sharing useful templates!
